xml/System.Runtime.InteropServices/UnmanagedCallersOnlyAttribute.xml

```diff
@@ -138,6 +138,7 @@ Supplied types must be from the official <xref:System.Runtime.CompilerServices>
 ## Remarks
 
 The export name must be a valid identifier for a native symbol on the target platform. The identifier must not conflict with system-provided native identifiers (for example, names of methods in the C standard library, or POSIX/Win32 API names).
+Using prefix for symbols is a common way to avoid name collisions of native symbols. For example, you can name your native symbols `mylibrary_free`, `mylibrary_isdigit` and `mylibrary_copy` to avoid name collisions.
 
 Target-dependent name mangling will be performed on the symbol name such as prepending or appending `_` or `@` characters to distinguish calling conventions on x86 Windows.
```

Comment:
```suggestion
Target-dependent name mangling will be performed on the symbol name such as prepending or appending `_` or `@` characters to distinguish calling conventions on x86.
```

These calling conventions are supported by x86 on any platform.

Reply:
Is the name mangling used for these calling conventions on non-Windows platforms?

Reply:
https://en.wikipedia.org/wiki/Name_mangling suggests that the name mangling is Windows specific.